### PR TITLE
Feature/story 17/active items

### PR DESCRIPTION
**Pull request by hale4029**
item's index page only showing active items - story 17

may need to extend class method for later stories

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -25,4 +25,7 @@ def no_orders?
     item_orders.empty?
   end
 
+  def self.find_active_items
+    where(active?: true)
+  end
```
Comment (mintona) [inline app/models/item.rb:30]: Is there a model test for this?

File: spec/features/items/index_spec.rb [hunk 2]
```diff
@@ -47,15 +47,23 @@
...
+
+      # within "#item-#{@dog_bone.id}" do
+      #   expect(page).to have_link(@dog_bone.name)
+      #   expect(page).to have_content(@dog_bone.description)
+      #   expect(page).to have_content("Price: $#{@dog_bone.price}")
+      #   expect(page).to have_content("Inactive")
+      #   expect(page).to have_content("Inventory: #{@dog_bone.inventory}")
+      #   expect(page).to have_link(@brian.name)
+      #   expect(page).to have_css("img[src*='#{@dog_bone.image}']")
+      # end
```
Comment (mintona) [inline spec/features/items/index_spec.rb:66]: From Harrison: This doesn't work now b/c dog bone is inactive so he took it out.

**Review by mintona — COMMENTED** [review]
Needs model test for model method before merging.

File: spec/features/items/index_spec.rb (hunk 2 above)
Comment (hale4029) [reply spec/features/items/index_spec.rb:66]: yes ma'am... nice catch, made.

File: spec/models/item_spec.rb [hunk 3]
```diff
@@ -47,5 +47,11 @@
       order.item_orders.create(item: @chain, price: @chain.price, quantity: 2)
       expect(@chain.no_orders?).to eq(false)
     end
+
+    it "finds all active pets" do
+      @bike_shop.items.create(name: "Dog Bone", description: "They'll love it!", price: 21, image: "https://img.chewy.com/is/image/catalog/54226_MAIN._AC_SL1500_V1534449573_.jpg", active?:false, inventory: 21)
```
Comment (mintona) [inline spec/models/item_spec.rb:52]: Thanks you!

**Review by mintona — APPROVED** [review]

**Comment by mintona** [comment]
I am good w/ this to merge into master. The next person who approves can merge it.

**Comment by DavidBarriga-Gomez** [comment]
Good methods for navigating items

**Pull request merged**